specification/app/resource-manager/Microsoft.App/preview/2023-04-01-preview/examples/Job_CreateorUpdate_EventTrigger.json

```diff
@@ -166,7 +166,7 @@
               "replicaCompletionCount": 1,
               "parallelism": 4
             },
```

Comment:
"eventTriggerConfig": {
        "parallelism": count,
        "replicaCompletionCount": count,
        "scale": {
          "maxReplicas": count,
          "minReplicas": count,
          "rules": [
            {
              "name": "rule_name",
              "custom": {
                "type": "azure-queue",
                "metadata": {
                  "customkey": "keyvalue"
                },
                "auth": []
              },
            }
          ]
        }
      }

Reply:
is this change really needed ? the current trigger type is supported no ?

Reply:
this is the response of the create call, so for a create call for a trigger type of 'event' with 'eventTriggerConfig' configuration, you would get a response of type 'Event' with configuration having 'eventTriggerConfig' not 'manualTriggerConfig'